bin/svg-sprite.js

```diff
@@ -3,17 +3,18 @@
 'use strict';
 
 /**
- * svg-sprite is a Node.js module for creating SVG sprites
+ * Svg-sprite is a Node.js module for creating SVG sprites
```

Comment:
Is there any way we don't touch the license headers?

Reply:
Yes, https://github.com/hosseinmd/prettier-plugin-jsdoc/#Options
`jsdocCapitalizeDescription: false`